### PR TITLE
Remove commas in filename

### PR DESCRIPTION
**Pull request by ohemorange**
This will immediately address the breakage reported in #6682 and tracked at #6685. Pip 19.0.0 and 19.0.1 don't allow commas in filenames, so don't use commas in filenames in certbot-apache test code.

I've confirmed that this fixes the issue on a machine that fails with the version of certbot-auto currently in master: recent version of virtualenv, python 2.7.

Steps to test:
- push master to test box
- run tools/venv.py
- activate venv
- pip --version: 19.0.1
- pip install ./certbot-apache/: fails
- push branch code to test box
- confirm pip --version still 19.0.1
- pip install ./certbot-apache/: success

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,13 +19,16 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 * Fixed accessing josepy contents through acme.jose when the full acme.jose
   path is used.
 * Clarify behavior for deleting certs as part of revocation.
+* Rename old,default.conf to old-and-default.conf to address breakages
+  from venv downloading the latest pip
```
Comment (bmw) [inline CHANGELOG.md]: nit: Can we generalize this and say something like:

> Rename old,default.conf to old-and-default.conf to address commas in filenames breaking recent versions of pip.

File: certbot-apache/certbot_apache/tests/testdata/debian_apache_2_4/augeas_vhosts/apache2/sites-enabled/old-and-default.conf [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<VirtualHost *:80 [::]:80>
```
Comment (bmw) [inline certbot-apache/certbot_apache/tests/testdata/debian_apache_2_4/augeas_vhosts/apache2/sites-enabled/old-and-default.conf]: It looks like this file got changed from a symlink to the file in `sites-available` to another copy of the file.

File: CHANGELOG.md (hunk 1 above)
Comment (ohemorange) [reply CHANGELOG.md]: Yep, great idea

File: certbot-apache/certbot_apache/tests/testdata/debian_apache_2_4/augeas_vhosts/apache2/sites-enabled/old-and-default.conf (hunk 2 above)
Comment (ohemorange) [reply certbot-apache/certbot_apache/tests/testdata/debian_apache_2_4/augeas_vhosts/apache2/sites-enabled/old-and-default.conf]: Hm, I didn't copy the file, I made a symlink. I'll look into that.

Comment (ohemorange) [reply certbot-apache/certbot_apache/tests/testdata/debian_apache_2_4/augeas_vhosts/apache2/sites-enabled/old-and-default.conf]: This sure is weird, it's definitely a copy of the file now

Comment (ohemorange) [reply certbot-apache/certbot_apache/tests/testdata/debian_apache_2_4/augeas_vhosts/apache2/sites-enabled/old-and-default.conf]: Oh I forgot the `-s` flag haaaa

**Pull request merged**